modules/hcp-deploy-openstack-parameters.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * hosted-control-planes/hcp-deploy/hcp-deploy-openstack.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="hcp-deploy-openstack-parameters_{context}"]
+= Options for creating a Hosted Control Planes cluster on OpenStack
+
+You can supply several options to the `hcp` CLI while deploying a Hosted Control Planes Cluster on {rh-openstack-first}.
+
+|===
+|Option|Description|Required
+
+|`--openstack-ca-cert-file`
+|Path to the OpenStack CA certificate file.
+|No
+
+|`--openstack-cloud`
+|Name of the cloud in `clouds.yaml`. The default value is `openstack`.
+|No
+
+|`--openstack-credentials-file`
+|Path to the OpenStack credentials file.
```

Comment:
Again, this shouldn't need to be specified now.

```suggestion
|Path to the OpenStack credentials file. If not provided, `hcp` will search the following directories: `.` (the current directory), `$HOME/.config/openstack`, `/etc/openstack`
```

Reply:
```suggestion
|Path to the OpenStack credentials file. If not provided, `hcp` will search the following directories:

* The current working directory
*`$HOME/.config/openstack`
* `/etc/openstack`
```